app/code/community/Elgentos/CodebaseExceptions/Model/Log/Writer/Stream.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class Elgentos_CodebaseExceptions_Model_Log_Writer_Stream extends Zend_Log_Writer_Stream
+{
+    /**
+     * Write a message to the log.
+     *
+     * @param  array  $event  event data
+     * @return void
+     * @throws Zend_Log_Exception
+     */
+    protected function _write($event)
```

Comment:
Could you please show me how to use this? A detailed explanation is appreciated :)